src/Service.Tests/SqlTests/RestApiTests/Find/MySqlFindApiTests.cs

```diff
@@ -414,7 +414,7 @@ LIMIT 100
             {
                 "FindTestWithPrimaryKeyContainingForeignKey",
                 @"
-                    SELECT JSON_OBJECT('id', id, 'book_id', book_id, 'content', content) AS data
+                    SELECT JSON_OBJECT('id', id, 'content', content) AS data
```

Comment:
two lines below this there is the statement:
```sql
SELECT id, book_id, content
```
Would it be valid to also remove book_id from the select statement here as you did for this outer select?